### ISSUE TITLE
Versions not shown (only dev-master)

### ISSUE THREAD

**Issue by sergwizard** [composer/packagist#836]
I registered my package in packagist.org and there on the page of the package it shows that - There is  latest version (dev-master) of this package.
How to change the dev-master to a version, for example, 1.0.3. . In the githab I have the appropriate tags and releases. But at the same time packagist.org shows the former (dev-master)?

**Comment by gregurco** [comment]
@sergwizard did you create "GitHub Service Hook"? Packagist don't do infinite check of new version, it should be pinged by github. 
1) open this link: https://packagist.org/about
2) read the section "GitHub Service Hook"

**Comment by stof** [comment]
Can you give the link to your package ? It would make debugging much easier.

**Comment by p5vgregurco** [comment]
I see only 2 packages from @sergwizard 
https://packagist.org/packages/sergwizard/yii2-geo-ip
https://packagist.org/packages/sergwizard/maxmind-geolite2-db
both have versions besides `dev-master`...

**Comment by gregurco** [comment]
I see only 2 packages from @sergwizard
https://packagist.org/packages/sergwizard/yii2-geo-ip
https://packagist.org/packages/sergwizard/maxmind-geolite2-db
both have versions besides `dev-master`...

**Comment by sergwizard** [comment]
I'm deceived for the answer with a delay. I solved the problem. The fact is that I did everything according to the instructions on the githab and packagist. The version in the packagist appeared only when I added the version again. That is, if I added the version in the repository, then I created the hook for parsing in packagist, then the version was not specified. But in the future, with the version added, everything worked fine.
